multi-doc-chatbot.py

```diff
@@ -8,28 +8,35 @@
 from langchain.vectorstores import Chroma
 from langchain.chat_models import ChatOpenAI
 from langchain.chains import ConversationalRetrievalChain
-from langchain.text_splitter import CharacterTextSplitter
+from langchain.text_splitter import RecursiveCharacterTextSplitter
 
 load_dotenv(".env")
 
 documents = []
 # Create a List of Documents from all of our files in the ./docs folder
-for file in os.listdir("docs"):
-    if file.endswith(".pdf"):
-        pdf_path = "./docs/" + file
-        loader = PyPDFLoader(pdf_path)
-        documents.extend(loader.load())
-    elif file.endswith(".docx") or file.endswith(".doc"):
-        doc_path = "./docs/" + file
-        loader = Docx2txtLoader(doc_path)
-        documents.extend(loader.load())
-    elif file.endswith(".txt"):
-        text_path = "./docs/" + file
-        loader = TextLoader(text_path)
-        documents.extend(loader.load())
+files_to_import = os.listdir("docs")
+num_files = str(len(files_to_import))
+for i, file in enumerate(files_to_import):
+    print("Importing: " + file + " (" + str(i + 1) + "/" + num_files + ")")
```

Comment:
```suggestion
    print(f"Importing: {file} {i+1} / {num_files}")
```

f strings is pretty cool syntactic sugar if you haven't seen it already